src/components/SocialBar/SocialBar.st.css

```diff
@@ -0,0 +1,28 @@
+
+
+/*Defaults*/
+:vars {
+  IconWidth: 24px;
+  IconsGap: 40px;
+  /*Defaults params goes here*/
+}
+
+/*Overrides*/
+:vars {
+  /*Deceleration of overridable placeholders*/
+}
+
+.root {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 20px 0; /* reserving space for copy url success text */
+}
+
+
+
+.item {
+  width: calc(value(IconWidth) + value(IconsGap));
+  justify-content: center;
+  display: flex;
```

Comment:
**Design review comment:**
Social bar gets squashed on mobile (see image).

I think we might need to add:
`flex-shrink: 0;`

In addition, we need to add `width: 100%` to the root, and an `overflow-x: auto`, to enable scrolling when there are many icons.

Since this is only for mobile, we might want to use the mobile property from the `TPAComponentsConsumer`.

![mobilesocial](https://user-images.githubusercontent.com/3277080/71666810-37415300-2d6b-11ea-8966-edcd1253aeed.png)